libcloud/storage/drivers/azure_blobs.py

```diff
@@ -156,6 +156,27 @@ class AzureBlobsConnection(AzureConnection):
     """
     Represents a single connection to Azure Blobs
     """
+    def __init__(self, *args, **kwargs):
+        self.account_prefix = kwargs.pop('account_prefix', None)
```

Comment:
consider adding 'account_prefix=None' as an argument, [rather than using kwargs](https://libcloud.readthedocs.io/en/latest/development.html#don-t-abuse-kwargs)

Reply:
Done.

Reply:
This was causing issues so reverted the change.